### PR TITLE
Update Woo Purple color palette

### LINKED ISSUE #12211: [Design review] Orders/products tab: update selected row background color
Design ref: w3JBReW1UJesCf02sj43kR-fi-1894_11886

Image

### PR DESCRIPTION
**Pull request by bozidarsevo**
Closes: #12211

## Description

Updating Woo Purple color palette based on p6riRB-8ox-p2

## Testing instructions

- Browse thru the app and check that app looks ok in general.
- Color changes are not big so it should not change app functionality a lot

## Screenshots

Change for colors (left side is old, right side is new):

woopurple

| Before      | After |
| ----------- | ----------- |
| Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-03-20 at 11 52 59      | Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-03-20 at 11 49 38       |

---
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by bozidarsevo** [comment]
@woocommerce/ios-developers it is not a big change but wanted to let everybody know so you are not surprised to see colors change :)

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@
 17.9
 -----
 - [Internal] Payments: Updated StripeTerminal pod to 3.3.1 [https://github.com/woocommerce/woocommerce-ios/pull/12078]
+- Update Woo Purple color palette [https://github.com/woocommerce/woocommerce-ios/pull/12330]
```
Comment (jaclync) [inline RELEASE-NOTES.txt]: There's no merge conflict, but I think we want to add this for release 18.0 in the latest trunk now

**Review by jaclync — APPROVED** [review]
Tested logging in & a few screens managing a store and the new colors look great! :shipit: 

Unrelated to this PR: maybe the selected cell background color in dark mode can be updated to have a better contrast with the table background color? The light mode color looks fine.

orders | products
-- | --
Screenshot 2024-03-25 at 4 46 57 PM | Screenshot 2024-03-25 at 4 49 00 PM

**Comment by bozidarsevo** [comment]
> have a better contrast

Yeah, makes sense, I think I will go with `shade10`:

Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-03-25 at 10 12 39

**Pull request merged**